### PR TITLE
Teams Section - BrickHack Club Site

### LINKED ISSUE #1211: Teams Section - BrickHack Club Site
Make teams section for #1206 

[Figma](https://www.figma.com/file/ZR9Mb4xNRrnoRZoX0bCWsX/BrickHack-Club-Website?node-id=0%3A1)

### PR DESCRIPTION
**Pull request by sjmiller7**
Fixes #1211 

The menu floats away if opened while the screen is resized but I don't have a solution right now. Can't figure out a jquery resize listener that works.

### CODE REVIEW

**Review by peterkos — APPROVED** [review]
Great work!

Additional design feedback:

CleanShot 2021-08-23 at 13 50 28@2x

File: sass/bhclub.scss [hunk 1]
```diff
@@ -265,6 +265,114 @@ section {
...
+    font-size: 1.25rem;
+    display: inline-block;
+    padding: 1rem calc(1.125rem + 42px) 1rem 1.125rem;
+    font-weight: 600;
+    cursor: pointer;
+}
+
+// Dropdown button when closed
+.ui-selectmenu-button-closed {
+    background: url("../assets/bhclub/select-arrow-blue.svg") no-repeat right;
```
[the remark this answers is not in the document]
Comment (sjmiller7) [reply sass/bhclub.scss:285]: Genuinely if you have an idea on how to get the arrow like the Figma without a background image that would be great to know

**Comment by sjmiller7** [comment]
> Explain club team structure

I thought it was fairly clear, but we may want someone outside the club to verify.

> Less padding

I understand that it looks a bit funky, but it's intentional. Each section is set to take up one screen height minimum (so they are different "pages"?), and the content on each section is centered vertically within that screen. If you click on the nav link instead of scrolling exactly where the title hits the top it looks correct. 

If we want to change this, we should check with design first

File: sass/bhclub.scss [hunk 2]
```diff
@@ -265,6 +265,114 @@ section {
...
+// Dropdown button
+.ui-selectmenu-button {
+    appearance: none;
+    outline: none;
+    width: max-content;
+    border: 2px solid $blue;
+    border-radius: 5px;
+    font-size: 1.25rem;
+    display: inline-block;
+    padding: 1rem calc(1.125rem + 42px) 1rem 1.125rem;
```
[the remark this answers is not in the document]
Comment (sjmiller7) [reply sass/bhclub.scss:278]: Split into two lines for readability and added comments in the fix

File: sass/bhclub.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterkos) [reply sass/bhclub.scss:286]: There's some hacky stuff like [triangles](https://css-tricks.com/snippets/css/css-triangle/) + a pseudo element for the box part but svg is a good workaround imo

**Comment by peterkos** [comment]
> Each section is set to take up one screen height minimum
> If we want to change this, we should check with design first
Valid; yeah it just looked a bit larger on this section than the others. We can address this l8r as it looks like a simple tweak and affects more than just this PR

**Review by peterkos — COMMENTED** [review]
LGTM!

**Review by peterkos — APPROVED** [review]
LGTM for real this time(tm)

**Pull request merged**